great_expectations_v1/compatibility/typing_extensions.py

```diff
@@ -2,6 +2,13 @@
 
 from typing import Any, Callable, TypeVar
 
+try:
+    from typing import (  # type: ignore[attr-defined]  # only exists in some python versions
+        Annotated,
+    )
+except ImportError:
+    from typing_extensions import Annotated
```

Comment:
Were these changes merged to `develop` as well?

Reply:
It was merged in this PR: https://github.com/great-expectations/great_expectations/pull/9994